src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -58,6 +58,10 @@ error.incorrect.length=\
         incorrect length while processing: {0}
 error.create.tempfile=\
         Could not create a temporary file
+error.extract.multiple.dest.dir=\
+        You may not specify more than one directory for extracting the jar
+error.extract.pflag.not.allowed=\
+        -P option cannot be used when extracting a jar to a specific location
```

Comment:
Perhaps something similar to

You may not specify '-Px'  with the '-C' or '--dir' options

Reply:
Makes sense. I've updated the PR with this change.